### ISSUE TITLE
French keyboard user can't enter backslack into editor

### ISSUE THREAD

**Issue by auchenberg** [Microsoft/vscode#41991]
From Twitter via DM: https://twitter.com/VincentSchoener

**Actual** 
When pressing `alt+shift+:` in VS Code, the "help top menu" opens. On french keyboard this is the shortcut for backslash.

![trvedijx](https://user-images.githubusercontent.com/173559/35238585-b80e4120-ff62-11e7-8f94-b2dd212c8f26.jpg)

**Expected**
Enter `\` into the editor

**Comment by vschoener** [comment]
That's me :) Thanks for the issue, I hope this will be resolved asap :)

**Comment by alexandrudima** [comment]
@vschoener We have a special purpose UI to help you manage keybindings. Just press `cmd+k cmd+s`.

There, you can search for "control+alt+shift+:" or "alt+shift+:". From your screenshot, that looks like some extension doing something with git.

---

![keyboard-conflicts](https://cloud.githubusercontent.com/assets/5047891/25484174/38f5b004-2b59-11e7-994b-dfa96ef043ce.gif)
